backend/code/m-booking/src/main/java/com/mbooking/controller/ReservationController.java

```diff
@@ -33,12 +33,12 @@
 	@Autowired
 	PaymentService paymentService;
 	
-	@GetMapping("day_expected_total_price/{id}")		//Test Manifestation day id
+	@GetMapping("day_expected_total_price/{id}")		//Test Manifestation day sectionId
```

Comment:
I'm afraid that renaming a dto variable took the toll on the comments as well. My bad, I will try to fix it 😓